include/Realisierung.tex

```diff
@@ -153,7 +153,86 @@ \subsection{Training}
 
 \subsection{Anwendung des Trainierten Modells}
 
-Um nun ein Bild zu analysieren und zu bestimmen wo sich die Personen im Bild befinden, wurde die Klasse \textit{People\_Finder} implementiert. Diese bietet nach aussen die Methode \textit{find\_people(image)} an. Diese Methode returniert die Treffer und deren Position. 
+Um nun ein Bild zu analysieren und zu bestimmen, wo sich die Personen im Bild befinden, wurde die Klasse \textit{People\_Finder} implementiert. Diese bietet nach aussen die Methode \textit{find\_people(image)} an. Diese Methode returniert die Treffer und deren Position.\\
+In dieser Methode wird das Bild gepadded und in Windows unterteilt, diese werden dann mit dem \gls{CNN} ausgewertet. Wird ein Window als Person markiert, wird das Zentrum des Windows gespeichert. Um mehrere Treffer auf derselben Person zu verhindern werden abschliessend alle Treffer geclustert. So werden Treffer die direkt nebeneinander liegen zu einem einzelnen Treffer zusammengeführt.
```

Comment:
In dieser Methode wird das Bild _mit Padding versehen_ und in Windows unterteilt, [...]
Eventuell für diese beiden Fachbegriffe einen Glossareintrag? Oder werden diese vorher im Detail erklärt?